HTML/HTML.sublime-syntax

```diff
@@ -15,22 +15,24 @@ variables:
   not_equals_lookahead: (?=\s*[^\s=])
 
   # https://html.spec.whatwg.org/multipage/parsing.html#tag-name-state
-  tag_name: '[[:alpha:]_][^/>\s]*'
+  tag_name_break: (?=[\s/>])
```

Comment:
I think that `\s` might be too broad. Only a handful of whitespace characters are prohibited: `[\t\n\f />]`.

Reply:
`\s` does include more?

Reply:
`unquoted_attribute_value` uses `\s` as well.

Reply:
Outside Unicode mode, `\s` also includes CR and vertical tab. In Unicode, it includes a lot more.